src/base/memory_monitor_sr.cpp

```diff
@@ -239,6 +241,38 @@ bool mmon_is_memory_monitor_enabled ()
   return (mmon_Gl != nullptr);
 }
 
+int mmon_initialize (const char *server_name)
+{
+  int error = NO_ERROR;
+
+  assert (mmon_Gl == nullptr);
+  assert_release (server_name != NULL);
+
+  if (prm_get_bool_value (PRM_ID_ENABLE_MEMORY_MONITORING))
+    {
+      mmon_Gl = new (std::nothrow) memory_monitor (server_name);
```

Comment:
To prevent abnormal termination of the program due to `bad_alloc` when memory allocation fails in memory_monitor because of memory shortage, and to incorporate it into the error handling process of CUBRID.

Reply:
I've got it. I didn't know this. Thank you.

Probably, we should use this way for other `new` as well at least from now on. The following kind of code doesn't work properly but just causes abnormal termination. Do I get it correct?
```
1218   /* Initialize the log block data buffer */
1219   /* *INDENT-OFF* */
1220   vacuum_Block_data_buffer = new lockfree::circular_queue<vacuum_data_entry> (VACUUM_BLOCK_DATA_BUFFER_CAPACITY);
1221   /* *INDENT-ON* */
1222   if (vacuum_Block_data_buffer == NULL)
1223     {
1224       goto error;
1225     }
```

Reply:
If `memory_monitor` is merged into the develop branch, then there is no need to use `std::nothrow` for new operators within the scope of `memory_monitor`. This is because if allocation fails, the return will proceed similarly to nothrow. However, the usage presented under the assumption of generally using new is incorrect.

Reply:
I understand that `nothrow` was likely used in this function to return an error code, enabling the caller to handle errors. Therefore, shouldn't this `nothrow` be retained even when merged into the develop branch?